### ISSUE TITLE
Accuracy

### ISSUE THREAD

**Issue by stephen-a-white** [noaa-rsd/LiProd#24]
https://github.com/noaa-rsd/LiProd/blob/0f4209e08ca5637f6243966554b12097d7d5fbb1/metadata_xml/metadata_template%20-%20Copy.xml#L95

Thoughts on if we should place "QL2B", equation, or if we have TPU (utilize to create a metric here), instead of  0.20m

**Comment by gimahori** [comment]
I think given the wide use of our data - we may want to consider doing both - have the TPU (and put the QL2B in parenthesis)?
In general hydro still doesn't use Quality level terminology but this maybe a good way to get them more comfortable with it and help those using the data who are familiar with it

**Comment by forkozi** [comment]
Not to muddy the waters (too much), but is there a particular thing the \<vertaccv\> field should describe.  Is it referring to how well the check points match up...or is it supposed to be more about the modeled uncertainty (i.e., the distribution of the cBLUE-calculated uncertainty values)?

**Comment by stephen-a-white** [comment]
It is a metric for the quality of the data.  In the past people have checked against survey control and provided an accuracy statement on how well it tested.  But since we have different options now, we could possibly answer that question with the TPU.  I think the distribution of the calculated uncertainty values would be good.  Probably should bounce the idea of how to report the uncertainty from TPU in metadata off a few people.

**Comment by gimahori** [comment]
agreed - I think we can probably have a good conversation with Jack Riley and Katrina/Glen... anyone else? Or bring it up at the meeting with Chris and Brian C.?

**Comment by stephen-a-white** [comment]
I think Chris P. and Brian would be good to talk with and get their thoughts.

**Comment by stephen-a-white** [comment]
After talking with Nick yesterday, maybe it makes sense to let the THU and TVU speak for itself.  That is why we are producing it.  Just need to figure verbiage for communicating it to public.

Maybe:
 GeoTiff Band 2 is Total Horizontal Uncertainty (THU), and Band 3 is Total Vertical Uncertainty (TVU)
